nodes/stampzilla-cloud/main.go

```diff
@@ -0,0 +1,384 @@
+package main
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"net/http/httputil"
+	"net/url"
+	"os"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/google/uuid"
+	"github.com/onrik/logrus/filename"
+	"github.com/sirupsen/logrus"
+	"github.com/stampzilla/stampzilla-go/nodes/stampzilla-server/models"
+	"google.golang.org/api/option"
+	"google.golang.org/api/transport"
```

Comment:
Missing generation and usage of `github.com/rakyll/statik`. Or is the html files embedded in some other way into the binary?